src/Network/Network/NetworkManager/NetworkManagerSecurityUserConfiguration/NetworkManagerSecurityUserCollection/SetAzNetworkManagerSecurityUserRuleCollectionCommand.cs

```diff
@@ -0,0 +1,117 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using AutoMapper;
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Commands.Network.Models.NetworkManager;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Management.Network;
+using System;
+using System.Management.Automation;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("Set", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetworkManagerSecurityUserRuleCollection", SupportsShouldProcess = true, DefaultParameterSetName = ByInputObject), OutputType(typeof(PSNetworkManagerSecurityUserRuleCollection))]
+    public class SetAzNetworkManagerSecurityUserRuleCollection : NetworkManagerSecurityUserRuleCollectionBaseCmdlet
+    {
+        private const string ByResourceId = "ByResourceId";
+        private const string ByInputObject = "ByInputObject";
+
+        [Alias("ResourceName")]
```

Comment:
Please provide Resource Name parameter set

Reply:
Clarification. Do I need to provide this parameter set for **Set**AzNetworkManagerSecurityUserRuleCollectionCommand.cs or **Get**AzNetworkManagerSecurityUserRuleCollectionCommand.cs?

Reply:
> Clarification. Do I need to provide this parameter set for **Set**AzNetworkManagerSecurityUserRuleCollectionCommand.cs or **Get**AzNetworkManagerSecurityUserRuleCollectionCommand.cs?

Yes, we should provide Resource Name parameter set for all the cmdlets